blocks/carousel/__track/_type/carousel__track_type_about.css

```diff
@@ -0,0 +1,6 @@
+/* this is how width is calculated: slide is 100% of the screen,
+     there are 2 slides, then width=2*100% */
+
+.carousel__track_type_about {
+  width: 200%;
```

Comment:
What will happen when an administrator adds a slide to this carousel? Is he need to change styles? Seems like a bad way